conans/test/functional/generators/link_order_test.py

```diff
@@ -144,7 +144,8 @@ def setUpClass(cls):
 
     def _validate_link_order(self, libs):
         # Check that all the libraries are there:
-        self.assertEqual(len(libs), 19 if platform.system() == "Darwin" else 16 if platform.system() == "Linux" else 26)
+        self.assertEqual(len(libs), 19 if platform.system() == "Darwin" else 16 if platform.system() == "Linux" else 26,
+                         msg="Unexpected number of libs ({}): '{}'".format(len(libs), "', '".join(libs)))
```

Comment:
There are some line-length violations in this file. I'm changing them here https://github.com/conan-io/conan/pull/6932